mc2.model.csv

```diff
@@ -47,6 +47,136 @@ Resource Access Hazard,"Sensory hazards (Flashing, Motion, etc.) applicable to t
+GeoMx DSP NGS Sequencing Platform,A platform is an object aggregate that is the set of instruments and software needed to perform a process [OBI_0000050]. Specific model of the sequencing instrument.,,,FALSE,,,,HTAN,
+GeoMx DSP NGS Library Selection Method,How RNA molecules are isolated.,,,FALSE,,,,HTAN,
+GeoMx DSP NGS Library Preparation Kit Name,Name of Library Preparation Kit. String,,,FALSE,,,,HTAN,
+GeoMx DSP Library Preparation Kit Vendor,Vendor of Library Preparation Kit. String,,,FALSE,,,,HTAN,
+GeoMx DSP Library Preparation Kit Version,Version of Library Preparation Kit. String,,,FALSE,,,,HTAN,
+NanoString GeoMx DSP Spatial Transcriptomics Level 2,Raw counts (DCC/RCC) files from GeoMX experiments,,"Component, Filename, File Format, Parent Biospecimen ID, Synapse ID of Data File, Synapse ID of Parent FASTQ File, Synapse ID of GeoMx Lab Worksheet File, Synapse ID of GeoMx DSP PKC File, Synapse ID of GeoMx DSP ROI Segment Annotation File, GeoMx DSP Assay Type, ROI Name, AOI Name, Software and Version, GeoMx DSP NGS Sequencing Platform, GeoMx DSP NGS Library Selection Method, GeoMx DSP NGS Library Preparation Kit Name, GeoMx DSP Library Preparation Kit Vendor, GeoMx DSP Library Preparation Kit Version",FALSE,,,,HTAN,
+Synapse ID of Parent FASTQ File,Synapse ID(s) indicating the FASTQ file(s) from which the DCC file was derived; multiple parent files should be comma-separated,,,FALSE,,,,HTAN,
+NanoString GeoMx DSP Imaging Level 2,,,"Component, Filename, File Format, Parent Biospecimen ID, Synapse ID of Data File, Synapse ID of GeoMx DCC File, Synapse ID of GeoMx ROI Segment Annotation File, ROI Name, AOI Name, Channel Metadata Filename, Imaging Assay Type, Protocol Link, Software and Version, Microscope, Objective, NominalMagnification, LensNA, WorkingDistance, WorkingDistanceUnit, Immersion, Pyramid, Zstack, Tseries, FOV number, FOVX, FOVXUnit, FOVY, FOVYUnit, Frame Averaging, Image ID, DimensionOrder, PhysicalSizeX, PhysicalSizeXUnit, PhysicalSizeY, PhysicalSizeYUnit, PhysicalSizeZ, PhysicalSizeZUnit, Pixels BigEndian, PlaneCount, SizeC, SizeT, SizeX, SizeY, SizeZ, PixelType",FALSE,,,,HTAN,
+Synapse ID of GeoMx DCC File,Synapse ID(s) for related DCC file(s) derived from the GeoMx DSP experiment.,,,FALSE,,,,,
+Channel Metadata Filename,Full path within Synapse project of uploaded companion CSV file containing channel-level metadata details,,,FALSE,,,,HTAN,
+Imaging Assay Type,Type of imaging assay,,,FALSE,,,,HTAN,
+Protocol Link,"Protocols.io ID or DOI link to a free/open protocol resource describing in detail the assay protocol (e.g. surface markers used in Smart-seq, dissociation duration,  lot/batch numbers for key reagents such as primers, sequencing reagent kits, etc.) or the protocol by which the sample was obtained or generated.",,,FALSE,,,,HTAN,
+Microscope,"Microscope type (manufacturer, model, etc) used for this experiment",,,FALSE,,,,HTAN,
```

Comment:
I would urge review of the microscope, lens, and magnification related fields for ST data. I imagine that the `platform`  will have fixed imaging configurations. So these fields may not be necessary to capture. The `platform` or the associated `imagingLabworksheet` will be the proxy for these metadata.

Reply:
I agree, many of these fields will be fixed, I'm still trying to find a source for many of the details.

I did find this GEO listing, suggesting the microscope uses 20x magnification: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM4844875